openapi/versions/v1.json

```diff
@@ -41,12 +41,11 @@
   },
   "servers": [
     {
-      "url": "https://API_GATEWAY_ID.execute-api.ap-south-1.amazonaws.com/api",
-      "description": "localstack"
+      "url": "https://BRANCH_NAME.bloodconnect.net/api"
     },
     {
-      "url": "https://${ENVIRONMENT}.bloodconnect.net/${API_VERSION}",
-      "description": "${ENVIRONMENT}"
+      "url": "https://API_GATEWAY_ID.execute-api.ap-south-1.amazonaws.com/api",
+      "description": "localstack"
```

Comment:
is this expected?

Reply:
also updated the domain url pattern to take the branch name from swagger setup script

Reply:
fair